### ISSUE TITLE
Extraneous precision

### ISSUE THREAD

**Issue by dsernst** [MaxGraey/react-native-console-time-polyfill#7]
### Expected: 

From Readme's Output section:

>  Example init: 200ms

### Actual:

I'm seeing an extra `.000` precision added.

<img src="https://user-images.githubusercontent.com/6340841/81528848-6c117800-9312-11ea-905a-8a0da072c6aa.png" width="300px" />

It looks like it's always reading exactly zero thousandths, even though this is timing a network call which has no reason to be that exact. In other words, I think it's not just unnecessary but incorrect.

### The code looks like this:

```ts
console.time('Example')
fetch(example).then((res) => {
  console.timeEnd('Example')
})
```

### Version info:

```
node: v12.16.3
macOS: 10.15

"react": "16.11.0",
"react-native": "0.62.0",
"react-native-console-time-polyfill": "^1.2.1",
```

**Comment by dsernst** [comment]
Looks like this is because of the `delta.toFixed(DEFAULT_PREC)` call. Is that necessary for something I'm not seeing?

Oh, I forgot to mention I'm running this in Xcode's iOS Simulator: `iOS 13.4.1`.

**Comment by MaxGraey** [comment]
Currently I use `time.toFixed(3)` and usually it's enough. But if you see always zeros after dot it really looks strange. But I guess simulator could have worsted precision than actual device.

**Comment by MaxGraey** [comment]
Fixed in 073f0216dd4e0506c3edca01890bc27786fddbf4
